File: Day 10 : Binary Numbers/Solution.c
#include <stdio.h>

int main()
{
    int n, count = 0, remainder = 0, no_of_1s = 0;
    
    scanf("%d", &n);
    
    while(n > 0)
    {
        if(n % 2 == 1)
        {
            count++;
            if(count > no_of_1s)
            {
                no_of_1s = count;
            }
        }
        else
        {
            count = 0;
            
        }
        n /= 2;
    }
    
    printf("%d", no_of_1s);
    
    return 0;
}

File: Day 10 : Binary Numbers/Solution.java
import java.util.*;

public class Solution
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        int count = 0, no_of_1s = 0;
        
        while(n > 0)
        {
            int remainder = n % 2;
            if(remainder == 1)
            {
                count++;
                if(count > no_of_1s)
                {
                    no_of_1s = count;
                }
            }
            else
            {
                count = 0;
            }
            n /= 2;
        }
        
        System.out.println(no_of_1s);
        scan.close();
    }
}
